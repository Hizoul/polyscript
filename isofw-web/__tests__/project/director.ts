import directorTest from "isofw-shared/src/tests/project/director"
import ShotEditor from "isofw-web/src/components/project/shotEditor";

directorTest(ShotEditor)
