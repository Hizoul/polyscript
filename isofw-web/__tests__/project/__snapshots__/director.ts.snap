// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` direct a test   can increase number :  info pushed by Server  1`] = `
<div
  className="flex1 column"
>
  <div
    className="flex center"
  >
    <div
      className="titleBox"
    >
      pn1
    </div>
  </div>
  <div
    className="flex center"
  >
    <div
      className="currentBox"
    >
      <span
        className="shotNumber"
      >
        42
      </span>
      <br />
      <span>
        CA ?
      </span>
    </div>
  </div>
  <div
    className="flex center"
  >
    <div
      className="Button"
    >
      {
  "text": "previous shot",
  "iconFa": "step-backward"
}
    </div>
    <div
      className="Button"
    >
      {
  "text": "next shot",
  "className": "flex1 center forwardButton",
  "iconFa": "step-forward"
}
    </div>
    <div
      className="Button"
    >
      {
  "iconFa": "sync",
  "text": "placeholder"
}
    </div>
  </div>
</div>
`;

exports[` direct a test   can increase number :  project Edited by client 1`] = `
Object {
  "createState": Object {},
  "getState": Object {
    "projects": Object {
      "b19999999999999999999999": Object {
        "result": Object {
          "_id": "b19999999999999999999999",
          "name": "pn1",
          "shot": 43,
        },
      },
    },
    "users": Object {
      "u1": Object {
        "result": Object {
          "_id": "u1",
          "email": "u1",
        },
      },
    },
  },
  "updateState": Object {
    "shotEdit.b19999999999999999999999projectModel": Object {
      "result": Object {
        "_id": "b19999999999999999999999",
        "name": "pn1",
        "shot": 43,
      },
    },
  },
}
`;

exports[` direct a test   can increase number :  project pushed by server  1`] = `
Object {
  "createState": Object {},
  "getState": Object {
    "projects": Object {
      "b19999999999999999999999": Object {
        "result": Object {
          "_id": "b19999999999999999999999",
          "name": "pn1",
          "shot": 42,
        },
      },
    },
    "users": Object {
      "u1": Object {
        "result": Object {
          "_id": "u1",
          "email": "u1",
        },
      },
    },
  },
  "updateState": Object {},
}
`;

exports[` direct a test   can increase number : After manual edits 1`] = `
<div
  className="flex1 column"
>
  <div
    className="flex center"
  >
    <div
      className="titleBox"
    >
      pn1
    </div>
  </div>
  <div
    className="flex center"
  >
    <div
      className="currentBox"
    >
      <span
        className="shotNumber"
      >
        43
      </span>
      <br />
      <span>
        CA ?
      </span>
    </div>
  </div>
  <div
    className="flex center"
  >
    <div
      className="Button"
    >
      {
  "text": "previous shot",
  "iconFa": "step-backward"
}
    </div>
    <div
      className="Button"
    >
      {
  "text": "next shot",
  "className": "flex1 center forwardButton",
  "iconFa": "step-forward"
}
    </div>
    <div
      className="Button"
    >
      {
  "iconFa": "sync",
  "text": "placeholder"
}
    </div>
  </div>
</div>
`;

exports[` direct a test   can increase number : Before manual edits 1`] = `
<div
  className="flex1 column"
>
  <div
    className="flex center"
  >
    <div
      className="titleBox"
    >
      pn1
    </div>
  </div>
  <div
    className="flex center"
  >
    <div
      className="currentBox"
    >
      <span
        className="shotNumber"
      >
        42
      </span>
      <br />
      <span>
        CA ?
      </span>
    </div>
  </div>
  <div
    className="flex center"
  >
    <div
      className="Button"
    >
      {
  "text": "previous shot",
  "iconFa": "step-backward"
}
    </div>
    <div
      className="Button"
    >
      {
  "text": "next shot",
  "className": "flex1 center forwardButton",
  "iconFa": "step-forward"
}
    </div>
    <div
      className="Button"
    >
      {
  "iconFa": "sync",
  "text": "placeholder"
}
    </div>
  </div>
</div>
`;
